install/operator/pkg/generator/assets/addons/dv/addon-dv-server.yml.tmpl

```diff
@@ -42,6 +42,12 @@
       metadata:
         labels:
           app: syndesis
+{{if .Productized}}
+          com.redhat.component-name: syndesis-dv
+          com.redhat.component-version: "{{.Tag}}"
+          com.redhat.product-name: "Red_Hat_Fuse"
+          com.redhat.product-version: "{{.ProductVersion}}"
+{{end}}
```

Comment:
These components should work in the same way as [amq-example](https://github.com/syndesisio/syndesis/blob/master/install/operator/pkg/generator/assets/infrastructure/04-amq-example.yml.tmpl#L1)